src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationServiceImpl.java

```diff
@@ -66,6 +79,46 @@ public Integer process(MultipartFile document, Annotation annotation) {
         return annotationEntity.getId();
     }
 
+    @Override
+    public AnnotationResponseDto downloadAnnotationDoc(Integer annotationId, Integer annotationDocumentId) {
+
+        final Optional<ExternalObjectDirectoryEntity> dirEntity = eodRepository.findAnnotationIdAndAnnotationDocumentId(annotationId, annotationDocumentId);
+        final InputStreamResource stream;
+
+        final ExternalObjectDirectoryEntity externalObjectDirectoryEntity;
+
+        if (dirEntity.isEmpty()) {
+
+            if (authorisationApi.userHasOneOfRoles(List.of(SecurityRoleEnum.JUDGE))) {
+                throw new DartsApiException(INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID_FOR_JUDGE);
```

Comment:
is there a benefit to the judge specific error message?

Reply:
This was how I interpreted the ticket. Do u think it incorrect?

Reply:
i've double checked with Dave, and there is no difference between Admin and Judge errors. i have updated the ACs to clarify